### PR TITLE
Add authorization for sender role

### PR DESCRIPTION
**Pull request by kft**
## Description
Add sender authorization

## References

* #0000

### CODE REVIEW

File: source/B2B.Transactions.CimMessageAdapter/Errors/AuthenticatedUserDoesNotHoldRequiredRoleType.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace B2B.CimMessageAdapter.Errors
+{
+    public class AuthenticatedUserDoesNotHoldRequiredRoleType : ValidationError
+    {
+        public AuthenticatedUserDoesNotHoldRequiredRoleType()
+            : base("The authenticated user does not hold the required.", "B2B-008")
```
Comment (Korgath) [inline source/B2B.Transactions.CimMessageAdapter/Errors/AuthenticatedUserDoesNotHoldRequiredRoleType.cs:20]: something got cut off in the message?

Comment (Mathiesen) [inline source/B2B.Transactions.CimMessageAdapter/Errors/AuthenticatedUserDoesNotHoldRequiredRoleType.cs:20]: Did you miss a word in that sentence?

Comment (kft) [reply source/B2B.Transactions.CimMessageAdapter/Errors/AuthenticatedUserDoesNotHoldRequiredRoleType.cs]: Fixed

Comment (kft) [reply source/B2B.Transactions.CimMessageAdapter/Errors/AuthenticatedUserDoesNotHoldRequiredRoleType.cs]: Fixed

**Review by Mathiesen — APPROVED** [review]

**Review by Korgath — APPROVED** [review]

**Comment by Korgath** [comment]
Ja tak - mega fedt

**Pull request merged**